src/utils/license.js

```diff
@@ -0,0 +1,32 @@
+/** @typedef {'cc0'|'by'|'by-sa'|'by-nc'|'by-nd'|'by-nc-nd'|'by-nc-sa'|'pdm'} License */
+
+/**
+ * Capitalized full license name from license slug and version:
+ * ('by-sa', '4.0') -> 'CC BY-SA 4.0'
+ * @param {string} license
+ * @param {string} license_version
+ * @return {string}
+ */
+export const getFullLicenseName = (license, license_version) => {
+  if (!license) {
+    return ''
+  }
+  const licenseUpper = license.toUpperCase()
+  return licenseUpper === 'CC0'
+    ? `${licenseUpper} ${license_version}`
+    : `CC ${licenseUpper} ${license_version}`
+}
+
+/**
+ * CC licenses have different legal status from the public domain marks
+ * such as CC0 and PDM, and need different wording.
+ * Returns false if licenseOrMark is CC0 or PDM
+ * @param {string} licenseOrMark
+ * @return {boolean}
+ */
+export const checkIsLicense = (licenseOrMark) => {
+  const licenseOrMarkLower = licenseOrMark ? licenseOrMark.toLowerCase() : ''
+  return (
+    !licenseOrMarkLower.includes('pdm') && !licenseOrMarkLower.includes('cc0')
+  )
+}
```

Comment:
Something like:

```js
export const isLicense = (licenseOrMark = '') => {
  return !['pdm', 'cc0'].includes(licenseOrMark.toLowerCase())
} 
```

```js
export const isPublicDomain = (licenseOrMark = '') => {
  return ['pdm', 'cc0'].includes(licenseOrMark.toLowerCase())
} 
```

might be a bit clearer? just one approach, i'm sure there's better.

Reply:
`isLicense` is confusing to me, but maybe it's a domain knowledge issue. Isn't `CC0` and `PDM` still a "license" even it's public domain?

Reply:
> Isn't CC0 and PDM still a "license" even it's public domain?

No actually! they are just "marks" or "designations", not licenses in the legal sense. CC even explains this quite poorly, admittedly. That's the purpose of this function, to determine if something is a license or a mark, so that front-end language can be changed.

Boring legal details: https://wiki.creativecommons.org/wiki/CC0_FAQ#What_is_CC0.3F

Reply:
Oh cool, that's good to know! Thanks for the explanation. In that case my comment doesn't really matter 🙂 Maybe we could add a link to that wiki page in the `@see` for the function to document for future contributors?

Reply:
Great question @sarayourfriend , and thank you for explanations, @zackkrida !
The CC Website has the following wording for its mission: "Provide **Creative Commons licenses** and **public domain tools** ...". 
I love the `isPublicDomain` function name, but would `isPDTool` be more correct in legal terms? I suppose that having a comment about difference between Licenses and Public Domain tools would be enough, though.